subsys/bluetooth/controller/hci_driver.c

```diff
@@ -347,6 +347,7 @@ static int hci_driver_open(void)
 			K_THREAD_STACK_SIZEOF(recv_thread_stack), recv_thread,
 			NULL, NULL, NULL, K_PRIO_COOP(CONFIG_BLECTLR_PRIO), 0,
 			K_NO_WAIT);
+	k_thread_name_set(&recv_thread_data, "blectlr recv");
```

Comment:
Maybe use ble_controller?

Reply:
There is a mix of `BLECTLR`, and `BLE_CONTROLLER` in use in this file elsewhere. I think this could be gone through and fixed up later :)